src/server/helpers/respond.js

```diff
@@ -22,6 +22,7 @@ export function respondWithError(
 }
 
 export function filterResponse(response, filterSchema) {
+  if (!response) return;
```

Comment:
What exactly is this preventing? Also, the return type of this method is an `Object`, and here it is `undefined`, it somehow smells like causing unexpected behaviour in other parts of the code 🕵️

Reply:
In the answers model, either propertyId or answerId isn't set (is null), but the association passes the null value through the filter functions and ends up here

Reply:
I would suggest prevent calling that method than if the value is null beforehands (I guess in filterResponseFields?), this method has a clear behaviour and this edge-case is not part of this method